jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/RemoteTaskServiceImpl.java

```diff
@@ -255,10 +255,14 @@ public void deleteTaskComment(String serverTemplateId,
         UserTaskServicesClient client = getClient(serverTemplateId,
                                                   UserTaskServicesClient.class);
 
-        List<TaskComment> comments = client.getTaskCommentsByTaskId(containerId,
-                                                                    taskId);
-
-        return comments.stream().map(c -> build(c)).collect(toList());
+        try {
+            List<TaskComment> comments = client.getTaskCommentsByTaskId(containerId,
+                                                                        taskId);
+            return comments.stream().map(c -> build(c)).collect(toList());
+        } catch (KieServicesException e) {
+            // task not found
```

Comment:
@nmirasch I think @bsremac got a point here, we could be hiding other errors with this approach. Is there any information in the _KieServicesException_ that we could make sure it is related to a _TaskNotFoundException_ only?